packages/main/src/plugin/progress-impl.spec.ts

```diff
@@ -78,6 +78,7 @@ test('Should create a task and propagate the exception', async () => {
 
   expect(updateTaskMock).toHaveBeenCalledTimes(1);
   expect(updateTaskMock).toHaveBeenCalledWith({
```

Comment:
I think the test 
```
  try {
    await progress.withProgress({ location: ProgressLocation.TASK_WIDGET, title: 'My task' }, async () => {
      throw new Error('dummy error');
    });
    // Should NEVER be here.
    expect(true).toBe(false);
  } catch (e: unknown) {
    expect((e as Error).message).toBe('dummy error');
  }

```

should use await expect(....).rejects.toThrow('dummy error') form

Reply:
Yes, I fixed it !